ccmlib/cluster.py

```diff
@@ -616,6 +616,13 @@ def drain(self):
     def repair(self):
         self.nodetool("repair")
 
+        stdout, _ = self.nodetool('help')
+        if " cluster:" in stdout:
+            for node in list(self.nodes.values()):
```

Comment:
isn't cluster repair, means I need todo it only once ?

i.e. what's the point of running it on all nodes ?

Reply:
also this code now does repair node by node, and then cluster repair node by node, 

I'm not sure it's the intention

Reply:
There is `break` after the first `cluster repair` on a running node

Reply:
but again, the code does regular reapir on all nodes, and then cluser repair....

Reply:
Yes, nodetool repair will only repair vnode keyspaces; nodetool cluster repair - only tablet keyspaces